pkg/controller/clusterautoscaler/clusterautoscaler_controller.go

```diff
@@ -78,16 +75,16 @@ func add(mgr manager.Manager, cfg *operator.Config, r reconcile.Reconciler) erro
 	// name set at runtime.
 	p := predicate.Funcs{
 		CreateFunc: func(e event.CreateEvent) bool {
-			return processEvent(e.Meta, cfg)
+			return r.NamePredicate(e.Meta)
```

Comment:
nit: it might read a little better if it was something like `r.NamesMatch)` - just seems more positive.

Reply:
Also, if nothing else uses `NamedPredicate()` you could just inline it here for locality: 

    predicate := func () ...